### PR TITLE
Feature/update performance

### PR DESCRIPTION
**Pull request by Azmat7860**
Sir please review the code. Thankyou

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -4,9 +4,11 @@ import express from "express";
 import userRoute from "./user/index.js";
 import attendenceRoute from "./attendence/index.js";
 import miscRoute from "./misc/index.js";
+import performanceRoute from "./performance/index.js";
 import eventRoute from "./event/index.js"
```
Comment (AbubakarZorrain) [inline routes/index.js:9]: Remove Empty Lines from import section...

File: routes/performance/controllers.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import PerformanceService from "../../services/performance.js";
+import httpResponse from "../../utils/httpResponse.js";
+
+const controller = {
+  add: async (req, res) => {
+    const addResponse = await PerformanceService.add(req.body);
+    if (addResponse.message === "success") {
+      return httpResponse.CREATED(res, addResponse.data);
```
Comment (AbubakarZorrain) [inline routes/performance/controllers.js:8]: Where is the status code for each request?

**Review by AbubakarZorrain — CHANGES_REQUESTED** [review]

File: routes/performance/controllers.js (hunk 2 above)
Comment (Azmat7860) [reply routes/performance/controllers.js:8]: Sir in httpResponse file all the status codes are mention there, we just call the function related to the specific status code.
e.g for status code 201 CREATED() method we are using.

**Comment by AbubakarZorrain** [comment]
Resolve Conflicts...

**Pull request merged**